src/telemetry/telemetryService.ts

```diff
@@ -119,6 +119,15 @@ export type ParticipantPromptProperties = {
   internal_purpose: InternalPromptPurpose;
 };
 
+export type ParticipantResponseProperties = {
+  command: string;
+  has_cta: boolean;
+  has_runnable_content: boolean;
+  found_namespace: boolean;
+  round_trips_number?: number; // TODO: how do we track this?
```

Comment:
Typically when adding a `TODO` in the code we have a ticket number with it, like `TODO(VSCODE-123): ...` that way we know either it's going to be resolved at some point, or if it's something like update a dependency, we can look at that ticket and see it's resolved and remove the TODO. 
Let's remove this until we decide if we want it or not?
```suggestion
```

Reply:
Yeah, I explicitly added it without a ticket to force us to make a decision on whether we want it or not during review. Seems like @GaurabAryal is in favor of dropping it for the initial release and revisiting after.